patroni/postgresql/config.py

```diff
@@ -942,24 +942,31 @@ def _get_tcp_local_address(self) -> str:
                 return 'localhost'  # connection via localhost is preferred
         return listen_addresses[0].strip()  # can't use localhost, take first address from listen_addresses
 
-    @property
-    def local_connect_kwargs(self) -> Dict[str, Any]:
-        ret = self._local_address.copy()
-        # add all of the other connection settings that are available
-        ret.update(self._superuser)
-        # if the "username" parameter is present, it actually needs to be "user"
-        # for connecting to PostgreSQL
-        if 'username' in self._superuser:
-            ret['user'] = self._superuser['username']
-            del ret['username']
-        # ensure certain Patroni configurations are available
-        ret.update({'dbname': self._postgresql.database,
-                    'fallback_application_name': 'Patroni',
-                    'connect_timeout': 3,
-                    'options': '-c statement_timeout=2000'})
-        return ret
-
     def resolve_connection_addresses(self) -> None:
+        """Calculates and sets local and remote connection urls and options.
+
+        This method sets:
+            * :attr:``Postgresql.connection_string`` attribute, which is later written to the member key in DCS
+              as ``conn_url``.
+            * :attr:``ConfigHandler.local_replication_address`` attribute, is used for replication connections
+              to local postgres.
+            * :attr:``ConnectionPool.conn_kwargs`` attribute, is used for superuser connections to local postgres.
```

Comment:
```suggestion
            * :attr:`Postgresql.connection_string <patroni.postgresql.Postgresql.connection_string>` attribute, which
              is later written to the member key in DCS as ``conn_url``.
            * :attr:`ConfigHandler.local_replication_address` attribute, which is used for replication connections to
              local postgres.
            * :attr:`ConnectionPool.conn_kwargs <patroni.postgresql.connection.ConnectionPool.conn_kwargs>` attribute,
              which is used for superuser connections to local postgres.
```

We need the full path to create links to things from other modules. In order to keep the alias, I suggested the syntax:

```
:attr:`ALIAS <PATH_TO_LINK>`
```